tests/integrationtest/r/explain-non-select-stmt.result

```diff
@@ -12,17 +12,15 @@ Insert	N/A	root		N/A
 explain format = 'brief' delete from t where a > 100;
 id	estRows	task	access object	operator info
 Delete	N/A	root		N/A
-└─SelectLock	3333.33	root		for update 0
```

Comment:
The is bug of the old mysql-tester, there should not be a SelectLock unless in the 'select ... for update' statement.